remotes/handlers.go

```diff
@@ -56,11 +56,22 @@ func FetchHandler(ingester content.Ingester, fetcher Fetcher) images.HandlerFunc
 func fetch(ctx context.Context, ingester content.Ingester, fetcher Fetcher, desc ocispec.Descriptor) error {
 	log.G(ctx).Debug("fetch")
 	ref := MakeRefKey(ctx, desc)
+
+	cw, err := ingester.Writer(ctx, ref, desc.Size, desc.Digest)
```

Comment:
@dmcgowan The problem is that it's hard to figure out on the `Dispatch` side, whether there is actually an error, or it's just a lock error, and I could start waiting and check progress.